### ISSUE TITLE
Set useWideViewPort to true for default Engine Settings

### ISSUE THREAD

**Issue by psymoon** [mozilla-mobile/firefox-tv#2504]
### Vision statement / What / Requirements
As per https://github.com/mozilla-mobile/firefox-tv/issues/2386 investigation, fullscreen mode on some websites are aligned poorly due to `useWideViewPort=false` in the `WebSettings`. Once https://github.com/mozilla-mobile/android-components/pull/3704 merges, Set `useWideViewPort=true` in the default Engine Setting.

### Impact
Site compatibility issues on some websites

### Risk
Unclear how setting `useWideViewPort=true` would affect other websites (especially the already working ones). WideViewPort is typically used for handling desktop mode (with user agent).

### Acceptance criteria
`useWideViewPort=true` is set in the default Engine Setting.

**Comment by athomasmoz** [comment]
@dnarcese Is this one already in progress?

**Comment by dnarcese** [comment]
@athomasmoz No, [the a-c issue ](https://github.com/mozilla-mobile/android-components/pull/3704) needs to merge before this can be started.

**Comment by psymoon** [comment]
It's merged now, now blocked on https://github.com/mozilla-mobile/firefox-tv/issues/2504
